### PR TITLE
Fixed #6787- SuiteCRM 7.11 User password not saving and could not change 

### LINKED ISSUE #5414: Password is not saving
#### Issue

In user profile, if the user pastes the password in the New Password field by right-clicking and pasting the value, the password will not be saved.

#### Expected Behavior

The password field value should be saved.

#### Actual Behavior

Password is not saving while creating and updating the password in user profile.

#### Steps to Reproduce

1. Go to the user profile.
2. Click on Create/Edit the user.
3. Fill the required fields
4. Copy the password from any file.
5. Right-click on the New Password Field.
6. Click the paste option.
7. Also, paste the value on the confirm password field by right-click.
8. Click Save.

#### Context

Low Priority

#### Your Environment

* SuiteCRM Version used: 7.10
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Version 60.0.3112.78 (Official Build) (64-bit)
* Environment name and version (e.g. MySQL, PHP 7): MySQL, PHP 5.5.9-1ubuntu4.21
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04

### LINKED ISSUE #6787: SuiteCRM 7.11  User password not saving and could not change
#### Expected Behavior

When creating new user In user profile, the password for its could not be saved or when I try to change password in Profile.
In USERS table field USER_HASH always is EMPTY.

#### Steps to Reproduce

1.
2.
3.
4.

#### Your Environment

* SuiteCRM Version used: SuiteCRM 7.11
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
* Environment name and version (e.g. MySQL, PHP 7):
* Operating System and version (e.g Ubuntu 16.04):

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

When creating a new user the password field isn't saved to the db.

## Motivation and Context

Issue reference: #6787

## How To Test This

1. Admin panel -> user management
2. Create a new user
3. Try to login as this user.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by jack7anderson7** [comment]
Assessed :+1:

**Comment by triggeroni** [comment]
Hi Dillon-Brown,

Just to comment on this in that I believe this issue is present in 7.10.12 also. Will this proposed fix be applied to the LTS version?

SuiteCRM Version used: 7.10.12
Browser name and version: Google Chrome Version 71.0.3578.98 (Official Build) (64-bit)
Environment name and version (e.g. MySQL, PHP 7): PHP 7.0 Server version: 10.1.26-MariaDB-0+deb9u1 Debian 9.1
Operating System and version (e.g Ubuntu 16.04): Debian Stretch

Thanks,

Tom.

**Comment by triggeroni** [comment]
Nevermind! https://github.com/salesagility/SuiteCRM/pull/6780

**Comment by jack7anderson7** [comment]
Assessed :+1:

File: modules/Users/User.php [hunk 1]
```diff
@@ -678,12 +670,18 @@ public function save($check_notify = false)
         // set some default preferences when creating a new user
         $setNewUserPreferences = empty($this->id) || !empty($this->new_with_id);
 
+        if (!$this->verify_data()) {
+            SugarApplication::appendErrorMessage($this->error_string);
+            header('Location: index.php?action=Error&module=Users');
```
Comment (gymad) [inline modules/Users/User.php]: Idk, is it possible here to use the proper function called SugarApplication::redirect() instead?

**Pull request merged**